modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -202,11 +215,44 @@ The Server Key is required to uniquely identify the server configuration. The re
 === Anypoint MQ Config
 
 [cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
-|======================
+|===
 | Field | Type | Description | Default Value | Required
 | Server Key | String | Identifier for the server configuration in AsyncAPI | None | Yes
 | Config Ref | String | Reference to the Anypoint MQ configuration details | None | No
-|======================
+|===
+
+[NOTE]
+The Server Key is required to uniquely identify the server configuration. Config Ref is optional and used to specify additional configuration details.
+
+[[SolaceConfig]]
+=== Solace Config
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|===
+| Field | Type | Description | Default Value | Required
+| Server Key | String | Server key set in the AsyncAPI spec | None | Yes
+| Config Ref | String | Reference to the configuration name of the Solace connection | None | No
+|===
+
+[NOTE]
+The Server Key is required to uniquely identify the server configuration. Config Ref is optional and used to specify additional configuration details.
+
+=== Limitations of Solace Connector
+
+The Solace connector in APIkit for AsyncAPI has the following limitations:
+
+* It doesn't support `topic listener`. It supports `queue listener`.
+* One channel to multiple queues isn't supported. A channel must have one queue. 
```

Comment:
This is not correct. The restriction is that you cannot have more than one operator of the same type mapped to the same destination queue. For instance, you cannot have more than one `Publish` operations mapped to the same destination queue.
@ashutoshsharmasf, please confirm.

Reply:
@ashutoshsharmasf, please, can you confirm that my rewording is appropriate? Thanks!